runtime/Cargo.toml

```diff
@@ -74,6 +74,8 @@ faucet = { path = "../pallets/faucet", default-features = false, optional = true
 hermes-governance-platform = { path = "../pallets/hermes-governance-platform", default-features = false }
 iroha-migration = { path = "../pallets/iroha-migration", default-features = false }
 iroha-migration-runtime-api = { path = "../pallets/iroha-migration/runtime-api", default-features = false }
+kensetsu = { path = "../pallets/kensetsu", default-features = false}
```

Comment:
It is better to make it optional while it is `wip` or `ready-to-test`. And add `wip` feature into Kensetsu's Cargo.toml.
In this case Kensetsu not even linked without `wip` flag